llvm/include/llvm/BinaryFormat/Wasm.h

```diff
@@ -350,6 +350,8 @@ struct WasmGlobal {
   WasmGlobalType Type;
   WasmInitExpr InitExpr;
   StringRef SymbolName; // from the "linking" section
+  uint32_t Offset;
+  uint32_t Size;
```

Comment:
Is this the size that the global itself occupies on the "global" section?  Might be worth a comment.

Reply:
Yes, added.